backend/cmd/migrate/migrate.go

```diff
@@ -33,6 +34,7 @@ func (m *MigrateFlags) Link() {
 	m.BaseFlags.Link()
 
 	flag.BoolVar(&m.Force, "f", false, "do not ask user for confirmation")
+	flag.BoolVar(&m.Down, "d", false, "migrates down by one version")
```

Comment:
let's make this a long-form arg -down if you don't mind
```suggestion
	flag.BoolVar(&m.Down, "down", false, "migrates down by one version")
```